### PR TITLE
[stable/kube-state-metrics]: Fix link to metric docu

### PR DESCRIPTION
**Pull request by florianrusch**
#### What this PR does / why we need it:
I've fixed the link to the metric docu in the NOTES.txt of the kube-state-metrics chart

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] title of the PR contains starts with chart name e.g. `[stable/chart]`

### CODE REVIEW

File: stable/kube-state-metrics/Chart.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ keywords:
 - metric
 - monitoring
 - prometheus
-version: 1.3.0
+version: 1.3.1
```
Comment (tariq1890) [inline stable/kube-state-metrics/Chart.yaml]:
```suggestion
version: 1.5.1
```

**Comment by tariq1890** [comment]
@florianrusch Could you rebase this PR?

**Comment by tariq1890** [comment]
@davidkarlsen FYI

**Comment by florianrusch** [comment]
@tariq1890 done

File: stable/kube-state-metrics/Chart.yaml [hunk 2]
```diff
@@ -5,7 +5,7 @@ keywords:
 - metric
 - monitoring
 - prometheus
-version: 1.3.0
+version: 1.5.1
```
Comment (tariq1890) [inline stable/kube-state-metrics/Chart.yaml]: This has to be changed again. Sorry :(.

```suggestion
version: 1.6.1
```

**Comment by tariq1890** [comment]
@florianrusch Thanks for your patience with this PR. Looks the following things need to be done.

i) Rebase with master and resolve merge conflicts.
ii) Fix the last review comment
iii) Squash the commits into one. The squash commit also needs to be signed off.

**Comment by florianrusch** [comment]
@tariq1890 Sorry, it's late here in germany. Hope everything is fine now :-)

**Pull request merged**